library/choose_disk.py

```diff
@@ -361,6 +363,17 @@ def disk_label(partition):
     if "ceph journal" in stdout:
         return "journal"
 
+    # Then, let's search for metadata coming from ceph-volume
+    output_cmd = subprocess.Popen("ceph-volume lvm list --format=json {}".format(partition),
+                                  shell=True, stdout=subprocess.PIPE)
+    raw_json, _ = output_cmd.communicate()
+    json = json.loads(raw_json)
+    for item in json:
+        current_item = json[item]
+        if "type" in current_item[0]:
+            return current_item[0]["type"]
+        else:
+            return "undefined"
```

Comment:
what would be the use case for having 'undefined' here vs. `""` below?

Reply:
I should have been more explicit. Let's me clarify that